tests/testapp/models.py

```diff
@@ -1,8 +1,23 @@
-# from django.db import models
+from django.db import models
+
 # from django.utils.translation import gettext_lazy as _
 
-from feincms3_meta.models import MetaMixin
+from feincms3_meta.fields import StructuredDataField, StructuredDataProperty
+from feincms3_meta.models import MetaMixin, StructuredDataMixin
 
 
 class Model(MetaMixin):
     pass
+
+
+class StructuredDataModel(StructuredDataMixin):
+    name = StructuredDataField(
+        models.CharField(max_length=50), "https://schema.org/name"
+    )
+
+    def get_absolute_url(self):
+        return "/slug/"
+
```

Comment:
It would be better if this was possible with a decorator, but I can't think of a solution.